src/bootstrap/dist.rs

```diff
@@ -1608,6 +1712,9 @@ impl Step for Extended {
                 if rls_installer.is_some() {
                     cmd.arg("-dRlsDir=rls");
                 }
+                if clippy_installer.is_some() {
+                    cmd.arg("-dRlsDir=clippy");
```

Comment:
I'm not actually sure what is happening here, but should this perhaps be `-dClippyDir`?